### PR TITLE
[M1566] make notes handling consistent and use SU notes for submit/ingest

### PR DESCRIPTION
**Pull request by saanobhaai**

### CODE REVIEW

**Comment by saanobhaai** [comment]
[audit and improve notes handling](https://trello.com/c/SWfqyswj/1566-audit-and-improve-notes-handling)

File: src/api/resources/sample_units/beltfishmethod.py [hunk 1]
```diff
@@ -326,8 +326,9 @@ class BeltFishMethodSUCSVSerializer(ReportSerializer):
         ReportField("biomass_kgha", "Biomass_kgha"),
         ReportField("biomass_kgha_by_trophic_group", "Biomass kgha by trophic group"),
         ReportField("site_notes", "Site notes"),
-        ReportField("sample_event_notes", "Sampling event notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/beltfishmethod.py:329]: Remove?

File: src/api/resources/sample_units/beltfishmethod.py [hunk 2]
```diff
@@ -373,7 +374,7 @@ class BeltFishMethodSECSVSerializer(ReportSerializer):
         ReportField("biomass_kgha_avg", "Biomass_kgha average"),
         ReportField("biomass_kgha_by_trophic_group_avg", "Biomass kgha by trophic group average"),
         ReportField("site_notes", "Site notes"),
-        ReportField("sample_event_notes", "Sampling event notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/beltfishmethod.py:377]: Remove

File: src/api/resources/sample_units/benthiclitmethod.py [hunk 3]
```diff
@@ -218,9 +217,10 @@ class ObsBenthicLITCSVSerializer(ReportSerializer):
         ReportField("length", "LIT (cm)"),
         ReportField("total_length", "Total transect cm"),
         ReportField("site_notes", "Site notes"),
-        # ReportField("sample_event_notes", "Sampling event notes"),
         ReportField("management_notes", "Management notes"),
-        ReportField("observation_notes", "Observation notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/benthiclitmethod.py:221]: Remove

File: src/api/resources/sample_units/benthicphotoquadrattransectmethod.py [hunk 4]
```diff
@@ -210,8 +210,9 @@ class BenthicPQTMethodSUCSVSerializer(ReportSerializer):
         ReportField("transect_len_surveyed", "Transect length surveyed"),
         ReportField("observers", "Observers", to_names),
         ReportField("site_notes", "Site notes"),
-        ReportField("sample_event_notes", "Sampling event notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/benthicphotoquadrattransectmethod.py:213]: Remove

File: src/api/resources/sample_units/benthicpitmethod.py [hunk 5]
```diff
@@ -191,9 +191,10 @@ class ObsBenthicPITCSVSerializer(ReportSerializer):
         ReportField("benthic_attribute", "Benthic attribute"),
         ReportField("growth_form", "Growth form"),
         ReportField("site_notes", "Site notes"),
-        # ReportField("sample_event_notes", "Sampling event notes"),
         ReportField("management_notes", "Management notes"),
-        ReportField("observation_notes", "Observation notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/benthicpitmethod.py:195]: Remove

File: src/api/resources/sample_units/bleachingquadratcollectionmethod.py [hunk 6]
```diff
@@ -243,6 +240,11 @@ class ObsBleachingQCColoniesBleachedCSVSerializer(ReportSerializer):
         ReportField("count_80", "50-80% bleached count"),
         ReportField("count_100", "80-100% bleached count"),
         ReportField("count_dead", "Recently dead count"),
+        ReportField("site_notes", "Site notes"),
+        ReportField("management_notes", "Management notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/bleachingquadratcollectionmethod.py:245]: Remove

File: src/api/resources/sample_units/habitatcomplexitymethod.py [hunk 7]
```diff
@@ -91,9 +91,10 @@ class ObsHabitatComplexityCSVSerializer(ReportSerializer):
         ReportField("score", "Habitat complexity value"),
         ReportField("score_name", "Habitat complexity name"),
         ReportField("site_notes", "Site notes"),
-        # ReportField("sample_event_notes", "Sampling event notes"),
         ReportField("management_notes", "Management notes"),
-        ReportField("observation_notes", "Observation notes"),
+        # ReportField("sample_event_notes", "Sampling event notes"),
```
Comment (gridcell) [inline src/api/resources/sample_units/habitatcomplexitymethod.py:95]: Remove

File: src/api/resources/summary_sample_event.py [hunk 8]
```diff
@@ -145,6 +157,7 @@ class SummarySampleEventCSVSerializer(ReportSerializer):
         ),
         ReportField("project_notes", "Project notes"),
         ReportField("site_notes", "Site notes"),
+        # ReportField("sample_event_notes", "Sample event notes"),
```
Comment (gridcell) [inline src/api/resources/summary_sample_event.py:160]: Remove

File: src/api/utils/summaries.py [hunk 9]
```diff
@@ -1,39 +1,14 @@
...
-        for record in summary_sites:
-            values = {
-                field.name: getattr(record, field.name)
-                for field in SummarySiteModel._meta.fields
-            }
-            SummarySiteModel.objects.create(**values)
-
-
 def update_project_summary_sample_event(project_id, skip_test_project=True):
+    print(project_id, skip_test_project)
```
Comment (gridcell) [inline src/api/utils/summaries.py:11]: remove

**Review by gridcell — CHANGES_REQUESTED** [review]

File: src/api/ingest/benthicpit_serializer.py [hunk 10]
```diff
@@ -54,16 +54,19 @@ class BenthicPITCSVSerializer(CollectRecordCSVSerializer):
         "Current": "data__benthic_transect__current",
         "Relative depth": "data__benthic_transect__relative_depth",
         "Tide": "data__benthic_transect__tide",
-        "Notes": "data__sample_event__notes",
+        "Sample unit notes": "data__benthic_transect__notes",
```
[the remark this answers is not in the document]
Comment (saanobhaai) [reply src/api/ingest/benthicpit_serializer.py:57]: I think it is... I can easily imagine a request for adding other notes fields...

**Review by gridcell — APPROVED** [review]

**Pull request merged**